docker/ipfs/config_ipfs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+ipfs config Addresses.API /ip4/0.0.0.0/tcp/5001
```

Comment:
Why did you choose this method over the ipfs config file one?
That's shame we have to modify two bash scripts, whereas using a config file seems to be built-in in the start script shipped with the container.